contracts/extensions/ERC721AOwnersExplicit.sol

```diff
@@ -10,7 +10,7 @@ error QuantityMustBeNonZero();
 error NoTokensMintedYet();
 
 abstract contract ERC721AOwnersExplicit is ERC721A {
-    uint256 public nextOwnerToExplicitlySet;
+    uint256 public nextOwnerToExplicitlySet = 0;
```

Comment:
You are undoing [this merged commit](https://github.com/chiru-labs/ERC721A/commit/11291663207e5679480204b4f1f3365e249bc868#diff-8a1021429bec822115cd1f665aabb3c58ce84d29f5d15ef16fae112ee093054eR9) here.

Reply:
Added back.